pkgs/applications/misc/pure-maps/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, mkDerivation, fetchFromGitHub, wrapQtAppsHook,
+qmake, qttools, kirigami2, qtquickcontrols2, qtlocation, qtsensors,
+nemo-qml-plugin-dbus, mapbox-gl-native, mapbox-gl-qml, s2geometry,
+python3, pyotherside, python3Packages
+}:
+mkDerivation rec {
+  pname = "pure-maps";
+  version = "2.2.0";
+  src = fetchFromGitHub {
+    owner = "rinigus";
+    repo = "pure-maps";
+    rev = version;
+    sha256 = "0xcrjzxnm3zx12ffb8yy87qa6iqga2h6ca6gi05q9xp5p6zg16j0";
+    fetchSubmodules = true;
```

Comment:
if we propagate gpxpy do we need to fetch the submodules?

Reply:
There's at least one other submodule, `geomag`, which is still required. In theory we could try to de-vendor them all, but I'm not sure it's trivial (for example, in one place, geomag is imported as `poor.geomag`, rather than as an external module).

Reply:
I think it would make sense to devendor gpxpy as it is already part of nixpkgs.